5calls/app/build.gradle

```diff
@@ -100,7 +94,7 @@ dependencies {
     implementation 'com.onesignal:OneSignal:5.1.28'
 
     // Plausible
-    implementation 'com.wbrawner.plausible:plausible-android:0.1.0-SNAPSHOT'
+    implementation 'com.github.OneBusAway:plausible-android-sdk:3.3'
 
     // If you add any other open-source libraries, make sure to update assets/licenses.html
     // with the appropriate license information.
```

Comment:
Do we need to update the licenses if we are using OneBusAway?

Reply:
Updated!